.dockerignore

```diff
@@ -0,0 +1,7 @@
+Dockerfile
+.gitignore
+.travis.yml
+*~
```

Comment:
oh, right, emacs leftovers in any directory you might touch a file.  Can emacs be configured to put all that stuff somewhere out of the way like /tmp ?